src/components/ec-currency-input/ec-currency-input.vue

```diff
@@ -106,6 +128,16 @@ export default {
     note: {
       type: String,
     },
+    bottomNote: {
+      type: String,
+    },
+    warningMessage: {
```

Comment:
@jafcalvente If this prop is for the tooltip, the name is not too clear to me, maybe it could be better to call it warningTooltipMessage?